src/app/shared/components/atoms/input-text/input-text.component.ts

```diff
@@ -1,21 +1,34 @@
 import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { ClassNames } from '@models/class-names';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
 
 @Component({
   selector: 'sk-input-text',
   templateUrl: './input-text.component.html',
   styleUrls: ['./input-text.component.scss'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      multi: true,
+      useExisting: InputTextComponent,
+    },
+  ],
 })
-export class InputTextComponent implements OnInit {
-  @Input() value = '';
+export class InputTextComponent implements OnInit, ControlValueAccessor {
+  @Input() value: string;
```

Comment:
An input value needs to be set for the value not to be `"undefined"`.

![image](https://user-images.githubusercontent.com/9085189/123039786-0b98a980-d3f3-11eb-93a9-751d943b9535.png)


```suggestion
  @Input() value: string = '';
```